internal/storage/activities/delete_amss.go

```diff
@@ -121,7 +147,7 @@ func (a *DeleteFromAMSSLocationActivity) requestDeletion(
 	ctx context.Context,
 	params *DeleteFromAMSSLocationActivityParams,
 	pipelineUUID string,
-) error {
+) (int64, error) {
```

Comment:
I think it would be helpful to name the return variables here to provide a hint about what the `int64` return represents (e.g. `id int64, err error`).